web/route.go

```
@@ -2,37 +2,58 @@ package web
+	"github.com/fenthope/sessions"
+	"github.com/fenthope/sessions/cookie"
 
-var (
-	logw       = logger.Logw
-	logInfo    = logger.LogInfo
-	logWarning = logger.LogWarning
-	logError   = logger.LogError
+	"github.com/infinite-iroha/touka"
 )
 
 var pagesPathRegex = regexp.MustCompile(`^[\w/]+$`)
```

golangci-lint (1.64.8)

var `pagesPathRegex` is unused

(unused)